### DISCUSSION TITLE
Problems with Capturing whole Student id

### DISCUSSION THREAD

**Discussion by BastiCoding** [community/community#128888] [GitHub Education] [answered]
Hi everyone,
I want to apply for GitHub Education i was a member in the past and Had no issues with the verification process but now with my new School my Student id hast two Sides which need to be both shown to reveal all information. At the verification process i can only Take one picture which doesnt Show my whole Info. If i apply with only one Side i get dennied because Not all information Is shown. Is there any way to submitt 2 Pictures for the verification so i can apply?

**Comment by emansarahafi** [comment, marked as answer]
I used CamScanner to scan both sides of my ID into one picture and then used that for the verification process. Otherwise, you can use any other documentation (transcripts, letters, etc.) with the necessary information to verify.

**Comment by Kr4k3n10** [comment]
Have the same problem with my ID and I'm unable to creat only one picture with CamScanner.

**Reply by emansarahafi** [reply]
Use the ID Cards feature on CamScanner, it will scan both sides of the card and show it on one page.
